openshift/manifests/02-serviceaccount-openshift-catalogd-controller-manager.yml

```diff
@@ -5,5 +5,5 @@ metadata:
   labels:
     app.kubernetes.io/name: catalogd
     app.kubernetes.io/part-of: olm
-  name: catalogd-controller-manager
+  name: controller-manager
```

Comment:
Seems weird that the `catalogd` prefix is removed here. Did you regenerate the manifests?